pkg/webhookadmission/webhook.go

```diff
@@ -182,9 +183,19 @@ func (c *Controller) checkVolumeOwner(volumes []v1.Volume, namespace string) (bo
 		if v.PersistentVolumeClaim == nil {
 			continue
 		}
-		pvc, err := core.Instance().GetPersistentVolumeClaim(v.PersistentVolumeClaim.ClaimName, namespace)
-		if err != nil {
-			return false, err
+
+		var pvc *v1.PersistentVolumeClaim
+		var err error
+		if storkcache.Instance() != nil {
+			pvc, err = storkcache.Instance().GetPersistentVolumeClaim(v.PersistentVolumeClaim.ClaimName, namespace)
+			if err != nil {
+				log.Errorf("failed to watch pods with informer cache for health monitoring, err: %v", err)
```

Comment:
1. We need to return error from here.
2. Error message doesn't seem right.

Reply:
done